docs/how-tos/hello-world-module.md

```diff
@@ -230,8 +258,87 @@ You need to add some sensor-specific code to support the sensor component.
 
     Save the <file>hello-world/src/main.py</file> file.
 
-1.  Open <file>temporary/meta.json</file> and copy the model information.
-    For example:
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+3. Edit the file structure:<br><br>
+
+   1. Change the name of <file>hello-world/models/module.go</file> to <file>hello-camera.go</file> and change the name of the <file>/models/</file> folder to <file>hellocamera</file> so you have <file>hello-world/hellocamera/hello-camera.go</file>.<br><br>
+
+   1. In the first line of <file>hello-camera.go</file>, change `package models` to `package hellocamera`.<br><br>
+
+   1. Change the name of <file>temporary/models/module.go</file> to <file>hello-sensor.go</file> and change the name of its folder to <file>hellosensor</file>.
+      Move the <file>hellosensor</file> folder from <file>temporary</file> to <file>hello-world</file> so you have <file>hello-world/hellosensor/hello-sensor.go</file>.<br><br>
+
+   1. In the first line of <file>hello-sensor.go</file>, change `package models` to `package hellosensor`.<br><br>
```

Comment:
We probably shouldn't tell them to update the directory names, but just the file names. So the file structure should be `hello-world/models/hello-camera.go` and `hello-world/models/hello-sensor.go`, both with the package name `package models`.

Reply:
What will the generator do when multi-model modules are added to its capabilities? Will it put them all in one models folder?

Reply:
I made all the necessary changes to get it to work but this way feels like more work for the user than making a directory per model, with the current way the files are generated. I think it'd be great if Config was named `mysensorConfig` or something by default.

Reply:
Likely when the generator can do multiple, it'll all be in the `models` directory